front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/notification/NotificationDAO.groovy

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.front50.model.notification
+
+interface NotificationDAO {
+  static final Collection<String> NOTIFICATION_FORMATS = ['sms', 'email', 'hipchat', 'slack']
```

Comment:
I just ripped it out of the Cassandra-specific `Repository`. 

I'm not sure it really adds much to be an `enum` except more LoC. 

TBH, Front50 is in an inconsistent state as far as coding styles go stemming back (_and possibly earlier_) to when 3 or 4 different services were squashed into one. 

If I had the time right now I'd nuke/deprecate all controllers and re-implement standard APIs on top of all the persistent entities ... but that will have to wait.